CHANGELOG.md

```diff
@@ -14,7 +14,7 @@
 ### Minor
 - Ensures proper removal of all `AircraftConflict` instances involving an aircraft that has been removed from the simulation [#133](https://github.com/n8rzz/atc/issues/133)
     - Originally reported under [zlsa#734](https://github.com/zlsa/atc/issues/734)
-
+- Changes the names from haveing the flags in their name  by adding WIP variable to the `AIRPORT_LOAD_LIST` in `airportLoadList` [#205](https://github.com/n8rzz/atc/issues/205)
```

Comment:
`having` is spelled incorrectly and there are two spaces between `name` and `by`